### DISCUSSION TITLE
creating custom profile

### DISCUSSION THREAD

**Discussion by niekgg** [Ericsson/codechecker#4309] [Q&A]
Hi everyone I want to create a custom profile where I will add list of analyzers and checkers to be enabled e.g. "my_custom_profile". Is it possible to achieve it?

**Comment by bruntib** [comment]
Hi, Profiles are managed in the label files found here: https://github.com/Ericsson/codechecker/tree/master/config/labels/analyzers-

**Reply by niekgg** [reply]
Thank you
